### ISSUE TITLE
Please include WCAG number in the eslint errors

### ISSUE THREAD

**Issue by ponienchen** [jsx-eslint/eslint-plugin-jsx-a11y#744]
When I run lint check, I could only see which a11y rules are violated, but it will be even helpful if it can also tell me which WCAG rules are violated.
